static/scripts/audit-report/audit.ts

```diff
@@ -306,7 +307,7 @@ class SmartQueue {
       if (git?.issue_number) {
         elemList.push({
           id: git.issue_number,
-          tx: ether?.txHash || "N/A", // @TODO - handle this better
+          tx: ether?.txHash || TX_EMPTY_VALUE, // @TODO - handle this better
```

Comment:
`null` is fine. And don't display `null` to the user.

Reply:
This one is not for display, it is the current default value when there is no `txHash` (just made it a variable to avoid typo mistakes)